### PR TITLE
Update token list to comply with new schema

### PR DESCRIPTION
**Pull request by elenadimitrova**
**Description**
Implement the token list schema standardisation proposed by Uniswap here https://github.com/Uniswap/token-lists/issues/51#issuecomment-952268629

**Metadata**
- Fixes https://linear.app/optimism/issue/INT-183/update-optimism-token-list-to-new-format

### CODE REVIEW

**Comment by willhennessy** [comment]
Hi Elena, thanks for the draft PR! When do you plan to release?

**Comment by elenadimitrova** [comment]
@willhennessy merging this depends on
- you agreeing this is final and merging https://github.com/Uniswap/token-lists/issues/51#issuecomment-952268629 
- @SmitV completing the gateway integration with the new schema and confirming this works

**Comment by willhennessy** [comment]
Great! Yes, talked to moody and we agree this is the right place to start for the extension. Thanks Elena

**Comment by elenadimitrova** [comment]
@moodysalem @willhennessy just fyi we are planning on using this format for L2 native tokens as well. Meaning an L2 native token entry will have the following setup
```
{
      "chainId": 10,
      ...
      "extensions": {
        "bridgeInfo": {
          "1": {
            "tokenAddress": "0xdeadbeef",
            "originBridgeAddress": "0xdeadbeef",
            "destBridgeAddress": "0xdeadbeef"
          }
        }
      }
    },
```

as compared to L1 native token entry
```
{
      "chainId": 1,
      ...
      "extensions": {
        "bridgeInfo": {
          "10": {
            "tokenAddress": "0xdeadbeef",
            "originBridgeAddress": "0xdeadbeef",
            "destBridgeAddress": "0xdeadbeef"
          }
        }
      }
    },
```

File: optimism.tokenlist.json [hunk 1]
```diff
@@ -16,18 +16,13 @@
...
-    {
-      "chainId": 10,
-      "address": "0x8700daec35af8ff88c16bdf0418774cb3d7599b4",
-      "name": "Synthetix",
-      "symbol": "SNX",
-      "decimals": 18,
-      "logoURI": "https://ethereum-optimism.github.io/logos/SNX.svg",
-      "extensions": {
-        "optimismBridgeAddress": "0x3f87Ff1de58128eF8FCb4c807eFD776E1aC72E51"
+        "bridgeInfo": {
```
Comment (justindomingue) [inline optimism.tokenlist.json:19]: i think dapps may expect to see `chainId: 10` at root level as opposed to only in `extensions`.

Comment (elenadimitrova) [reply optimism.tokenlist.json:19]: Sure, this is possible for L2 native tokens like these https://github.com/ethereum-optimism/ethereum-optimism.github.io/blob/2138386277e4156d159615d1840882cecc398437/optimism.tokenlist.json#L300
If a token is L1 native though it makes more sense its root node in the schema here to be for `chainId: 1`. It's more optimal normalisation of the data as opposed to the flat schema option.

**Review by 12Thee — APPROVED** [review]

**Pull request closed without merging**